test/integration_test.sh

```diff
@@ -28,159 +23,298 @@
+printf "\nss2_mode: $ss2_mode"
+printf "\nss2_version: $ss2_version"
+printf "\nenv_config_json: $env_config_json"
+printf "\nsecrets_json: $secrets_json"
+
+printf "\n\nWorking directory: $work_dir"
+printf "\nScript directory: $script_dir"
 
-# 2. Get pipeline-tools version
-# TODO: Uncomment condition below once we have Lira reading wrapper WDLs from pipeline-tools repo
-#if [ $pipeline_tools_mode == "github" ] && [ $pipeline_tools_version == "current" ]; then
-  pipeline_tools_version=$(python $script_dir/current_deployed_pipeline_version.py \
-                    --mint_deployment_dir $mint_deployment_dir \
+# 1. Clone mint-deployment
+printf "\n\nCloning mint-deployment\n"
+git clone git@github.com:HumanCellAtlas/mint-deployment.git
+mint_deployment_dir=mint-deployment
+
+# 2. Clone Lira if needed
+if [ $lira_mode == "github" ]; then
+  printf "\n\nCloning lira\n"
+  git clone git@github.com:HumanCellAtlas/lira.git
+  lira_dir=lira
+  cd $lira_dir
+  if [ $lira_version == "latest_released" ]; then
+    printf "\nDetermining latest release tag\n"
+    lira_version=$(python $script_dir/get_latest_release.py --repo HumanCellAtlas/lira)
+  elif [ $lira_version == "latest_deployed" ]; then
+    printf "\nDetermining latest deployed version\n"
+    lira_version=$(python $script_dir/current_deployed_version.py \
+                    --component_name lira
                     --env $env \
-                    --pipeline_name pipeline_tools)
-  echo "pipeline_tools_version: $pipeline_tools_version"
-#fi
+                    --mint_deployment_dir $mint_deployment_dir)
+  fi
+  printf "\nChecking out $lira_version\n"
+  git checkout $lira_version
+  cd $work_dir
+elif [ $lira_mode == "local" ]; then
+  printf "\n\nUsing Lira in dir: $lira_version\n"
+  lira_dir=$lira_version
+fi
 
-# 3. Build or pull Lira image
+# 3. Get pipeline-tools version
+if [ $pipeline_tools_mode == "github" ]; then
+  if [ $pipeline_tools_version == "latest_released" ]; then
+    printf "\n\nDetermining latest released version of pipeline-tools\n"
+    pipeline_tools_version=$(python $script_dir/get_latest_release.py --repo HumanCellAtlas/pipeline-tools)
+  elif [ $pipeline_tools_version == "latest_deployed" ]; then
+    printf "\n\nDetermining latest deployed version of pipeline-tools\n"
+    pipeline_tools_version=$(python $script_dir/current_deployed_version.py \
+                      --mint_deployment_dir $mint_deployment_dir \
+                      --env $env \
+                      --component_name pipeline_tools)
+  fi
+  printf "\nConfiguring Lira to use adapter wdls from pipeline-tools GitHub repo, version: $pipeline_tools_version\n"
+  pipeline_tools_prefix="https://raw.githubusercontent.com/HumanCellAtlas/pipeline-tools/${pipeline_tools_version}"
+elif [ $pipeline_tools_mode == "local" ]; then
+  pipeline_tools_prefix="/pipeline-tools"
+  pipeline_tools_dir=$pipeline_tools_version
+  cd $pipeline_tools_dir
+  pipeline_tools_dir=$(pwd)
```

Comment:
I don't understand why ` cd $pipeline_tools_dir` and `pipeline_tools_dir=$(pwd)` here are necessary if we already have `pipeline_tools_dir=$pipeline_tools_version`

Reply:
Sorry that's not clear - it gets the absolute path, which is required later for mounting it into the container. Will add a comment.